File: modul-4/PRAK302-2210817210012-MuhammadRakaAzwar.py
i = 1
while i < 2:
    num = input()
    num = int (num)
    if (num <0 and num >100):
        print ("Nilai tidak valid.")
    elif (num <=79 and num >=70):
        print ("B")
    elif (num <=69 and num >=60):
        print ("C")
    elif (num <=59 and num >=50):
        print ("D")
    elif (num <= 49):
        print ("E")
    else:
        print ("A")

File: modul-4/PRAK302-2210817210012-MuhammadRakaAzwar.c
#include <stdio.h>
int main()
{
    int num;
    int i = 1;
    while (i < 2){
    printf ("");
    scanf ("%d", &num);
    if (num <0 || num >100) {
            printf("Nilai tidak valid.\n");
    }
    else if (num <=79 && num >=70) {
            printf("B\n");
    }
    else if (num <=69 && num >=60) {
        printf ("C\n");
    }
    else if (num <=59 && num >=50) {
        printf ("D\n");
    }
    else if (num <= 49) {
        printf ("E\n");
    }
    else
        printf ("A\n");
    }
}
